runtime/layers/tests.php

```diff
@@ -0,0 +1,57 @@
+<?php /** @noinspection ALL */
+declare(strict_types=1);
+
+/**
+ * This file runs tests on Docker images.
+ */
```

Comment:
I added tests for Docker images. That helps check that some things are correctly configured.